.github/workflows/sync-redirects.yaml

```diff
@@ -0,0 +1,30 @@
+name: Sync RTD redirects
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - docs/redirects.yaml
+      - .github/workflows/sync-redirects.yaml
+
+  # Manually triggered using GitHub's UI
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+
+      - name: Upgrade Python toolchain
+        run: python3 -m pip install --upgrade pip setuptools wheel
+
+      - name: Install readthedocs-cli
+        run: python3 -m pip install readthedocs-cli
+
+      - name: Sync redirects
+        run: rtd projects nextstrain-augur redirects sync -f doc/redirects.yaml --wet-run
```

Comment:
```suggestion
        run: rtd projects nextstrain-augur redirects sync -f docs/redirects.yaml --wet-run
```

Reply:
Done in [force-push](https://github.com/nextstrain/augur/compare/c8607cb633ff516b17b5c3abd9a9170bfccc2806..50425d962939f5511878b067201ddd83c61d889b).